tests/integration/utils.py

```diff
@@ -31,7 +34,13 @@ def wait_for(predicate: Callable[[], bool], timeout: float = 10, sleep_time: flo
 
 
 IS_WINDOWS = os.name == "nt"
-TIMEOUT_MULTIPLIER = 2 if IS_WINDOWS else 1
+
+
+def is_windows_ci() -> bool:
```

Comment:
I think we can wrap everything in a `duration_threshold(0.2)`, so that we can just write everywhere in our tests:

```python
assert 0 <= duration <= duration_threshold(0.2), f"Duration is {duration} seconds"
```